package.json

```diff
@@ -23,11 +23,13 @@
     "test:ci": "npm run test -- --ci",
     "cypress": "cypress open",
     "cypress:headless": "cypress run",
+    "cypress:percy": "percy exec -- cypress run",
     "e2e": "start-server-and-test 'npm run build && npm run serve' http://localhost:3000 'npm run cypress'",
     "e2e:dev": "start-server-and-test 'npm run dev' http://localhost:3000 'npm run cypress'",
     "e2e:headless": "start-server-and-test 'npm run build && npm run serve' http://localhost:3000 'npm run cypress:headless'",
+    "e2e:percy": "start-server-and-test 'npm run build && npm run serve' http://localhost:3000 'npm run cypress:percy'",
     "gen:theme-typings": "chakra-cli tokens src/customTheme.ts --strict-component-types",
-    "prepare": "husky install",
+    "prepare": "is-ci || husky install",
```

Comment:
Just a small improvement to husky so we don't install it on CI, which should speed up a bit the dependencies installation.